### PR TITLE
Add known issue for image deletion stuck

### PR DESCRIPTION
**Pull request by WebberHuang1118**
Add known issue for image deletion stuck and workaround

### CODE REVIEW

File: versioned_docs/version-v1.1/upload-image.md [hunk 1]
```diff
@@ -64,4 +64,17 @@ You can add labels to the image, which will help identify the OS type more accur
 
 If your image name or URL contains any valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
 
-![](/img/v1.1/image-labels.png)
\ No newline at end of file
+![](/img/v1.1/image-labels.png)
+
+### Known Issues
+
+If Image is removed during uploading, there is a slim chance that the related Longhorn Backing Image get stuck in deletion.
```
Comment (connorkuehl) [inline versioned_docs/version-v1.1/upload-image.md:71]:
```suggestion
If an Image is removed during uploading, there is a slim chance that the related Longhorn Backing Image gets stuck in deletion.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 2]
```diff
@@ -64,4 +64,17 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+From [Longhorn Dashboards](./troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards), once you found matched Longhorn Backing Image keeps in deletion state for more than 5 minutes , the corner case hit.
```
Comment (connorkuehl) [inline versioned_docs/version-v1.1/upload-image.md:73]:
```suggestion
To see if a Longhorn backing image is affected with this issue, find it on the [Longhorn dashboard](./troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). If it has been stuck in a deletion state for more than 5 minutes, follow the workaround instructions linked below.
```

**Review by connorkuehl — COMMENTED** [review]
Just some minor tweaks, but let's defer to @jillian-maroket 

Otherwise LGTM.

**Comment by WebberHuang1118** [comment]
> Just some minor tweaks, but let's defer to @jillian-maroket
> 
> Otherwise LGTM.

Thanks @connorkuehl,
this PR is still under processing, I will update it with your suggestion and re-request your review.

File: versioned_docs/version-v1.1/upload-image.md (hunk 1 above)
Comment (WebberHuang1118) [reply versioned_docs/version-v1.1/upload-image.md]: Updated per suggestion, thanks.

File: versioned_docs/version-v1.1/upload-image.md (hunk 2 above)
Comment (WebberHuang1118) [reply versioned_docs/version-v1.1/upload-image.md]: Updated per suggestion, thanks.

File: versioned_docs/version-v1.1/upload-image.md [hunk 3]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
 
 If your image name or URL contains any valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
 
-![](/img/v1.1/image-labels.png)
\ No newline at end of file
+![](/img/v1.1/image-labels.png)
+
+### Known Issues
+
+If an Image is removed during uploading, there is a slim chance that the related Longhorn Backing Image gets stuck in deletion.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:71]:
```suggestion
If you remove an image before the upload process is completed, the related Longhorn backing image may become stuck in the deletion state.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 4]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+To see if a Longhorn backing image is affected with this issue, find it on the [Longhorn dashboard](./troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). If it has been stuck in a deletion state for more than 5 minutes, follow the workaround instructions below.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:73]:
```suggestion
To determine if the issue has occurred, locate the backing image on the Dashboard screen of the [Longhorn UI](./troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). If the deletion state has lasted for more than 5 minutes, perform the following steps:
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 5]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+![](/img/v1.1/delete-image-stuck.png)
+
+1. Find backingimagedatasource was deleted longer than 5 minutes ago:
+
+    `$ kubectl get backingimagedatasources.longhorn.io -A -o json | jq -r '.items[] | select(.metadata.deletionTimestamp != null) | .metadata.name + " deleted at: " + .metadata.deletionTimestamp '`, e.q.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:79]:
```suggestion
    `$ kubectl get backingimagedatasources.longhorn.io -A -o json | jq -r '.items[] | select(.metadata.deletionTimestamp != null) | .metadata.name + " deleted at: " + .metadata.deletionTimestamp '`
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 6]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+    ![](/img/v1.1/check-image-stuck.png)
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:81]:
```suggestion
    Example:
    ![](/img/v1.1/check-image-stuck.png)
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 7]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+### Known Issues
+
+If an Image is removed during uploading, there is a slim chance that the related Longhorn Backing Image gets stuck in deletion.
+
+To see if a Longhorn backing image is affected with this issue, find it on the [Longhorn dashboard](./troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). If it has been stuck in a deletion state for more than 5 minutes, follow the workaround instructions below.
+
+![](/img/v1.1/delete-image-stuck.png)
+
+1. Find backingimagedatasource was deleted longer than 5 minutes ago:
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:77]:
```suggestion
1. Verify that the deletion state has lasted for more than 5 minutes.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 8]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+    `$ kubectl get backingimagedatasources.longhorn.io -A -o json | jq -r '.items[] | select(.metadata.deletionTimestamp != null) | .metadata.name + " deleted at: " + .metadata.deletionTimestamp '`, e.q.
+
+    ![](/img/v1.1/check-image-stuck.png)
+
+2. Edit the backingimagedatasource which found from the above step:
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:83]:
```suggestion
1. Edit the affected backing image data source.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 9]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+    `$ kubectl -n longhorn-system edit backingimagedatasources.longhorn.io <backingimagedatasource name>`
+
+    e.q. $ kubectl -n longhorn-system edit backingimagedatasources.longhorn.io default-ubuntu-bionic
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:87]:
```suggestion
    Example:
    $ kubectl -n longhorn-system edit backingimagedatasources.longhorn.io default-ubuntu-bionic
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 10]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+3. Delete finalizer of the backingimagedatasource:
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:89]:
```suggestion
1. Delete the `finalizers` metadata of the affected backing image data source.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 11]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+   Delete the related two lines `finalizers:` and `- longhorn.io`
+
+   ![](/img/v1.1/delete-stuck-image-finalizer.png)
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:93]:
```suggestion
   Example:
   ![](/img/v1.1/delete-stuck-image-finalizer.png)
   You must delete `finalizers:` and `- longhorn.io`.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 12]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
...
+
+4. You should find the backingimage no longer stuck in deletion.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:95]:
```suggestion
The backing image should no longer be stuck in the deletion state.
```

File: versioned_docs/version-v1.1/upload-image.md [hunk 13]
```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
 
 If your image name or URL contains any valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.1/upload-image.md:65]:
```suggestion
The Harvester UI can extract information such as OS type and file format from image names and URLs. You can also manually specify the same information on the UI.
```

Comment (jillian-maroket) [reply versioned_docs/version-v1.1/upload-image.md:65]: I rewrote this paragraph for the Harvester PoC Guide. I'd like to apply the change here as well.

**Review by jillian-maroket — COMMENTED** [review]
Let me know if the changes work. Thanks!

File: versioned_docs/version-v1.1/upload-image.md (hunk 13 above)
Comment (WebberHuang1118) [reply versioned_docs/version-v1.1/upload-image.md:65]: @jillian-maroket 
I think maybe it's better to create another PR for this. 
WDYT? @bk201, thanks.

Comment (bk201) [reply versioned_docs/version-v1.1/upload-image.md:65]: Yeah, I think it's better to create another PR and include all supported versions.

File: versioned_docs/version-v1.1/upload-image.md [hunk 14]
```diff
@@ -64,4 +64,40 @@ You can add labels to the image, which will help identify the OS type more accur
...
+1. Verify that the deletion state has lasted for more than 5 minutes.
+
+    `$ kubectl get backingimagedatasources.longhorn.io -A -o json | jq -r '.items[] | select(.metadata.deletionTimestamp != null) | .metadata.name + " deleted at: " + .metadata.deletionTimestamp '`
+
+    Example:
+    ![](/img/v1.1/check-image-stuck.png)
+
+1. Edit the affected backing image data source.
+
+    `$ kubectl -n longhorn-system edit backingimagedatasources.longhorn.io <backingimagedatasource name>`
```
Comment (bk201) [inline versioned_docs/version-v1.1/upload-image.md:86]: Let's use the code block for commands. It's more consistent across docs.

File: versioned_docs/version-v1.1/upload-image.md (hunk 13 above)
Comment (jillian-maroket) [reply versioned_docs/version-v1.1/upload-image.md:65]: @WebberHuang1118 @bk201 In that case, I'll save this for later. We can create a PR for non-release content enhancements.

**Comment by WebberHuang1118** [comment]
Updated per suggestions, thanks.

**Review by jillian-maroket — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Review by connorkuehl — APPROVED** [review]

**Pull request merged**